cmd/sts-handlers.go

```diff
@@ -302,8 +303,9 @@ func (sts *stsAPIHandlers) AssumeRoleWithSSO(w http.ResponseWriter, r *http.Requ
 	if token == "" {
 		token = r.Form.Get(stsWebIdentityToken)
 	}
+	accessToken := r.Form.Get(stsWebIdentityAccessToken)
```

Comment:
I tested this against our gitlab instance, but the `accessToken` is always empty. Looking at `r.Form` it only contains the `WebIdentityToken` not the `WebIdentityAccessToken`.

Reply:
@bachp you need to provide just like `id_token` for JWT - accessToken needs to be provided if UserInfo() support is needed.

Reply:
@harshavardhana By provided you mean it should be returned by the OpenID connect provider? Or is there something I need to configure on minio side?

Reply:
@harshavardhana Is this something that needs to be implemented on minio/console side?

Reply:
MinIO console won't support this just yet, it is only implemented as an API support right now.

Userinfo endpoint cannot use the same id_token for authorization, it needs to be access_token unfortunately.

This is why this API is an extension and is only an opt in. If UserInfo is requested apps have to change

Reply:
Ok thanks for clarifying.